File: news-angular/src/app/model/AuthorModel.ts
import { TagsModel } from './TagsModel';

export class AuthorModel {
  id: String= "" ;
  tags: Array<TagsModel>= new Array() ;
  slug: String= "" ;
  username: String= "" ;
  password: String= "" ;
  firstName: String= "" ;
  lastName: String= "" ;
  dateBirth: Date ;
  city: String= "" ;
  country: String= "" ;
}

File: news_rest/src/main/java/com/example/trade_centre/model/AuthorModel.java
package com.example.trade_centre.model;

import lombok.Data;

import java.time.LocalDate;
import java.util.List;

@Data
public class AuthorModel {
    private String id;
    private List<TagsModel> tags;
    private String slug;
    private String username;
    private String password;
    private String firstName;
    private String lastName;
    private LocalDate dateBirth;
    private String city;
    private String country;
}
